### ISSUE TITLE
My Order page

### ISSUE THREAD

**Issue by valdrinkoshi** [hmecookd/HomeCooked#23]
- [x] handle case when there is no order placed
- [ ] show order information: dish name with link + price
- [ ] show pickup time & address

**Comment by dbaq** [comment]
what about showing previous orders?

**Comment by valdrinkoshi** [comment]
That is a nice to have, right @marcandreoli?

**Comment by dbaq** [comment]
I am thinking also about a legal standpoint here.

**Comment by marcandreoli** [comment]
Yes, good idea to show past orders here. We can show 'past orders' below live orders under a different 'past orders' line or tab.
For each past dish, need to show:
-date of order
-meal name
-meal qty
-meal price
-name of chef
We can sort them by date. (same as GrubHub)

**Comment by valdrinkoshi** [comment]
Of course is a good idea, I wanted to know if it's a must have for alpha release. I would do it only if it's a legal obligation

**Comment by marcandreoli** [comment]
It is a legal obligation to send a receipt - by email or inside the app. 
Yes, this is a 'nice to have'. Not an absolute necessity for Alpha. 
@valdrinkoshi You can decide based on the additional work, but I think it would be a nice 'nice to have'
